code/game/atoms_movable.dm

```diff
@@ -80,10 +80,10 @@
 		last_move = 0
 		return
 
+	last_move = direct

```

Comment:
Be careful with this, it can break diagonal movement depending on where it is located.